commons/protos/policy_interface.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+message Graph {
+  message Node {
+    string version = 1;
+    string image = 2;
+    map<string, string> metadata = 3;
+  }
+
+  message Edge {
+    uint64 from = 1;
+    uint64 to = 2;
+  }
+
+  repeated Node nodes = 1;
+  repeated Edge edges = 2;
+}
+
+message PluginInput {
+  Graph graph = 1;
+  map<string, string> parameters = 2;
+}
+
+message PluginOutput {
+  message Error {
+    enum Status {
+      SUCCESS = 0;
```

Comment:
Two comments on this:
 * I think we don't strictly need the "success" bit here if we can model the two arms of Result directly (see my comment below)
 * for JSON side, I picked up a textual `kind` field: https://github.com/openshift/cincinnati/pull/58/files#diff-1bb65de438c6b2131604dcc53ca74722R50. Do we want to align on a single approach?